### ISSUE TITLE
Event-target-shim breaks Create React App builds

### ISSUE THREAD

**Issue by Antman261** [mysticatea/event-target-shim#11]
I have an app bootstrapped with create-react-app and decided to use [abort-controller](https://www.npmjs.com/package/abort-controller) with [github's fetch polyfil](https://github.com/github/fetch).

As you know, abort-controller has this package as it's only dependency. When I attempt to build my package (Equivalent with .babelrc `{ "presets": ["react-app"]}`) I get the following:

```
Failed to compile.

Failed to minify the code from this file:

 	./node_modules/event-target-shim/dist/event-target-shim.js:61
```

I'd really like to use abort controller.

**Comment by mysticatea** [comment]
Thank you for this issue.

I guess that you used a minifier which doesn't support ES2015, then it was thrown at ES2015 syntax. Please try to transpile both abort-controller and event-target-shim also by Babel.

**Comment by braco** [comment]
+1, this is the only module that doesn't build out of a few hundred.

**Comment by lsiden** [comment]
@Antman261 and @braco, I found two workarounds for this but neither one is satisfying:

1. Copy abort-controller.js, abort-signal.js and event-target-shim.js to your own project and change the import statements to match.
2. eject and comment out or delete the initialization of webpack.optimize.UglifyJsPlugin in you webpack.config.production.

I don't like either workaround. (2) makes the output bundle twice as big.  

The basic problem is that it was written in ES6 and needs to be transpiled to ES5 to meet CRA's expectation for everything under node_modules/.  An easy fix is to transpile the code to ES5, but the authors already set it up this way and changing it might break it for others that are already using it.

Feel free to share any thoughts.  abort-controller et al are highly useful modules for some apps and I'd like to be able to use them in other projects with less resistance.

**Comment by lsiden** [comment]
Sounds good, but both the master branch and the latest release of react-scripts (packaged with with create-react-app) is still importing webpack@3.8.1.  Until then, I have no choice but to suffice with one of the workarounds I mentioned.

**Comment by fadomire** [comment]
same issue here but i'm running Webpack 4
this is the only node_module that is not already transpiled to ES5

**Comment by GeorgeTaveras1231** [comment]
I believe this problem will be fixed by #23

**Comment by mysticatea** [comment]
I had intended this package is used along with bundlers, and if you want to support IE11, you should configure your bundlers to transpile application.

But, as mentioned, some build tools don't look transpiling node_modules.
Therefore, since 6.0.0, this package includes ES5 build. Use `import {} from "event-target-shim/es5"` instead.
